source/_patterns/05-components/tag/tag.twig

```diff
@@ -1,9 +1,10 @@
 {% set classes = [
   'tag',
-  type ? 'tag--' ~ type,
-  size ? 'tag--' ~ size
+  modifier_classes ? modifier_classes,
 ]|join(' ')|trim %}
 
 <a href="{{ url }}" class="{{ classes }}" rel="tag">
-  {{ title }}
+  {% include "@uswds/tag/tag.twig" with {
+    text: title
+  }%}
 </a>
```

Comment:
How do you feel about renaming this component to “Tag Link” to make it more obvious what this component is used for?

Reply:
👍 That's a better name